src/org/intellij/erlang/ErlangReferenceContributor.java

```diff
@@ -16,43 +16,82 @@
 
 package org.intellij.erlang;
 
+import com.intellij.patterns.PsiElementPattern;
 import com.intellij.psi.*;
+import com.intellij.util.ObjectUtils;
 import com.intellij.util.ProcessingContext;
+import org.intellij.erlang.psi.ErlangArgumentList;
+import org.intellij.erlang.psi.ErlangFunctionCallExpression;
+import org.intellij.erlang.psi.ErlangMaxExpression;
 import org.intellij.erlang.psi.ErlangQAtom;
 import org.intellij.erlang.psi.impl.ErlangPsiImplUtil;
+import org.intellij.erlang.psi.impl.ErlangPsiImplUtil.ErlangFunctionCallModuleParameter;
+import org.intellij.erlang.psi.impl.ErlangPsiImplUtil.ErlangFunctionCallNamedParameter;
+import org.intellij.erlang.psi.impl.ErlangPsiImplUtil.ErlangFunctionCallParameter;
 import org.jetbrains.annotations.NotNull;
 
 import static com.intellij.patterns.PlatformPatterns.psiElement;
 
 public class ErlangReferenceContributor extends PsiReferenceContributor {
-  private enum Kind {FUNCTION, MODULE, RECORD}
-
   @Override
   public void registerReferenceProviders(@NotNull PsiReferenceRegistrar registrar) {
-    register(registrar, Kind.RECORD,   "erlang", "is_record",  2, 1);
-    register(registrar, Kind.RECORD,   "erlang", "is_record",  3, 1);
-    register(registrar, Kind.MODULE,   "erlang", "spawn",      3, 0);
-    register(registrar, Kind.FUNCTION, "erlang", "spawn",      3, 1);
-    register(registrar, Kind.MODULE,   "erlang", "spawn_link", 3, 0);
-    register(registrar, Kind.FUNCTION, "erlang", "spawn_link", 3, 1);
```

Comment:
We haven't got these functions in our mock SDK-s, but we have spawn_monitor/3 in one of mocks which is identical. Test added.